assets/javascripts/discourse/initializers/discourse-reactions.gjs

```diff
@@ -220,13 +177,93 @@ function initializeDiscourseReactions(api) {
   }
 }
 
+function customizePostMenu(api, container) {
+  const currentUser = container.lookup("service:current-user");
+
+  const transformerRegistered =
+    currentUser?.use_glimmer_post_menu &&
```

Comment:
Gotta think about what will happen to this logic after we remove the `use_glimmer_post_menu` boolean in future 🤔 

Could we just register the transformer unconditionally? It'll be a no-op if the user/site is still using the widget implementation?

Reply:
This logic is not needed anymore. 

This was from an earlier commit of the API